src/main/java/uk/gov/ons/census/casesvc/service/CaseService.java

```diff
@@ -214,6 +214,8 @@ private CollectionCase createCollectionCase(Case caze, Address address) {
     collectionCase.setCollectionExerciseId(caze.getCollectionExerciseId());
     collectionCase.setId(caze.getCaseId().toString());
     collectionCase.setSurvey(caze.getSurvey());
+    collectionCase.setCreatedDateTime(caze.getCreatedDateTime());
+    collectionCase.setLastUpdated(caze.getLastUpdated());
```

Comment:
`lastUpdated` isn't in the event dictionary (yet?) so should go in the comment block below.